x11-drivers/xf86-video-ast/xf86-video-ast-1.1.5.ebuild

```diff
@@ -6,6 +6,6 @@ EAPI=5
 inherit xorg-2
 
 DESCRIPTION="X.Org driver for ASpeedTech cards"
-KEYWORDS="amd64 x86 ~amd64-fbsd ~x86-fbsd"
+KEYWORDS="amd64 ppc ppc64 x86 ~amd64-fbsd ~x86-fbsd"
```

Comment:
You should not add straight to stable keywords for a package you've just tested. So it should be 

`KEYWORDS="amd64 ~ppc ~ppc64 x86 ~amd64-fbsd ~x86-fbsd"`

Reply:
The reason for that, as noted in the commit message, was that the CI blocks the unmasking of the video_cards_ast use-flag if there is no copy of this driver available in stable. 

Going straight to stable seemed the least worst option, as this driver is already stable on amd64 and x86.

Reply:
Surely it is better than making X ~ppc64, IMHO.